rotkehlchen/db/filtering.py

```diff
@@ -495,7 +495,7 @@ def make(
             and_op=True,
             from_ts=from_ts,
             to_ts=to_ts,
-            timestamp_attribute='time',
+            timestamp_attribute='timestamp',
```

Comment:
With this change can't we get rid of the `timestamp_attribute` argument? Since it should now **always** be `'timestamp'`?